assets/css/style-article.css

```
@@ -0,0 +1,2250 @@
+h1.title+.tagline+.content {
+    margin-top: 1.8rem;
+}
+.content pre {
+    white-space: normal;
+}
+.content hr {
+    border: 0;
+    height: 1px;
+    background-image: -webkit-linear-gradient(left, #f0f0f0, #5FADDA, #f0f0f0);
+    background-image: -moz-linear-gradient(left, #f0f0f0, #5FADDA, #f0f0f0);
+    background-image: linear-gradient(left, #f0f0f0, #5FADDA, #f0f0f0);
```

Biome (1.9.4)

[error] Unexpected nonstandard direction

You should fix the direction value to follow the syntax.
See MDN web docs for more details.

(lint/correctness/noInvalidDirectionInLinearGradient)